spaces/S000168/README.md

```diff
@@ -0,0 +1,22 @@
+---
+uid: S000168
+name: Real projective plane
+aliases:
+  - $\mathbb RP^2$
+  - $P_2(\mathbb R)$
+  - RP2
+refs:
+  - wikipedia: Real projective plane
```

Comment:
Need the underscores to match https://en.wikipedia.org/wiki/Real_projective_space (Steven's recommendation).